app/views/juke/index.html.erb

```diff
@@ -1,46 +1,47 @@
-<div class="site-wrapper">
-  <div class="site-wrapper-inner">
-    <div class="cover-container">
-      <div class="masthead clearfix">
-        <div class="inner">
-          <h3 class="masthead-brand">예수인교회 Jukebox</h3>
-          <nav>
-            <ul id='juke_control' class="nav masthead-nav">
-              <li>
-                <span data_url="/juke/previous" class="glyphicon glyphicon-backward" aria-hidden="true"></span>
-              </li>
-              <li id="play_button_container">
-                <%= render partial: 'play_button' %>
-              </li>
-              <li>
-                <span data_url="/juke/next" class="glyphicon glyphicon-forward" aria-hidden="true"></span>
-              </li>
-              <li>
-                <span data_url="/juke/stop" class="glyphicon glyphicon-stop" aria-hidden="true"></span>
-              </li>
-              <li>
-                <span data_url="/juke/clear" class="glyphicon glyphicon-eject" aria-hidden="true"></span>
-              </li>
-            </ul>
-          </nav>
-        </div>
-      </div>
+<section id="header" class="dark">
```

Comment:
Pro-tip: we have an unspoken rule about making 2 space indentation between blocks of code. You'll need to change your IDE setting to make this a default.